Sources/PointFree/Emails/NewEpisodeEmail.swift

```diff
@@ -10,8 +10,10 @@ import Prelude
 import Styleguide
 
 let newEpisodeEmail = simpleEmailLayout(newEpisodeEmailContent)
-  .contramap { ep, isSubscriber in
+  .contramap { ep, user, isSubscriber in
     SimpleEmailLayoutData(
+      user: user,
+      newsletter: .newEpisode,
```

Comment:
here's a case where we send the user and a newsletter to the layout so that it can render the footer properly.